.github/workflows/ci.yml

```diff
@@ -91,6 +117,18 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           args: --check runtime/
 
+      - if: "!cancelled()"
+        name: uncrustify
+        run: |
+          exit_value=0
+          for i in $(find ./src/nvim -name "*.[ch]"); do
+            if ! ${{ env.CACHE_UNCRUSTIFY }} -c ./src/uncrustify.cfg -q --check "$i"; then
+              echo "::error::$i isn't correctly formatted."
```

Comment:
this is definitely good enough for now, but is there an upstream issue we can track (and mention here)? Pretty weird that the example given by `man uncrustify`, doesn't work...

Reply:
Oh, maybe should've left a comment in the file about this: the example in `man uncrustify` does work. It prints all files it runs through and reports which one passes and which ones fails. The problem is that it's *really* hard to see which files fail since we have around 400 files we format. This approach will 1. print in the error message in red color so you can immediately spot it and 2. add a note about this in the summary page.

Reply:
I can simulate an error if you want a quick look at it?

Reply:
Sure. But then we'll miss out the full logs of which files have been uncrustifies ;).

I'll take your comment as this being an acceptable tradeoff.